source/CIGameScene.cpp

```diff
@@ -108,6 +108,9 @@ bool GameScene::init(const std::shared_ptr<cugl::AssetManager>& assets, bool isH
     _stardustContainer->addStardust(dimen);
     _stardustContainer->addStardust(dimen);
     
+    // TODO: resize to number of players in the game and add opponent planet nodes to scene graph
+    _opponent_planets.resize(5);
```

Comment:
should we have like a variable/constant somewhere that stores this?

Reply:
I don't think so because the 5 is just temporary because we should be risizing with the number of players actually in the game